apps/include/system/binary.h

```diff
@@ -63,14 +63,24 @@ int binary_update_get_binary_info(char *binary_name, binary_info_t *binary_info)
 
 /**
  * @brief Get the all binaries information
- * @details @b #include <apps/system/binary_update.h>\n
+ * @details @b #include <apps/system/binary.h>\n
  *  It requests the binary manager to get the all of binaries' information.
  * @param[out] binary_info_list The address value to receive the binary information list
  * @return A defined value of binmgr_response_result_type in <tinyara/binary_manager.h>
  *         0 (BINMGR_OK) On success. On failure, negative value is returned.
  * @since TizenRT v3.0
  */
 int binary_update_get_binary_info_all(binary_info_list_t *binary_info_list);
+
+/**
+ * @brief Notify that loaded binary is started to binary manager
+ * @details @b #include <apps/system/binary.h>\n
+ *  It sends a message the binary manager to notify that loaded binary is started well.
+ * @return A defined value of binmgr_response_result_type in <tinyara/binary_manager.h>
```

Comment:
could you check return type?(int or binmgr_response_result_type)
Also could you check the other APIs?